### ISSUE TITLE
Latest version of cloudwatch 

### ISSUE THREAD

**Issue by Vignesh-Kirubanandam** [micrometer-metrics/micrometer#3254]
The problem with me was that I was using micrometer-registry-cloudwatch2, which wasn't compatible with my SpringBoot. and Micronaut So I used micrometer-registry-cloudwatch and everything worked.

**Comment by jonatan-ivanov** [comment]
What do you mean by micrometer-registry-cloudwatch2 is not being compatible with SpringBoot or Micronaut?

Could you please give us some details (description about what you are trying to do; what error you get or issue bump into; a more descriptive title might help too)?

**Comment by Vignesh-Kirubanandam** [comment]
Hi Jonatan,

How can we log micronaut local level loggers into AWS cloudwatch? So then I
can share the exact logs.

Policy given to log cloudwatch.

"logs:CreateLogGroup", "logs:CreateLogStream", "logs:PutLogEvents",
"logs:DescribeLogStreams"

But how can we include the below logs
23:35:54.113 [pool-3-thread-6] DEBUG i.m.c.CloudWatchMeterRegistry -
published metric with namespace:Custom-Namespace

Thanks,
Vignesh

**Comment by shakuzen** [comment]
I'm not familiar with configuring logs for Micronaut applications on AWS and it's a bit out-of-scope for what we can troubleshoot/support here in Micrometer.

For Micronaut, are you using the dependency mentioned in [the documentation](https://micronaut-projects.github.io/micronaut-micrometer/latest/guide/#metricsAndReportersCloudWatch) for Micrometer Cloudwatch integration?

**Comment by jonatan-ivanov** [comment]
Since this seems to be coped to logging with Micronaut, let me close this issue.
Please let us know that it is not the case and you want us to reopen.

**Comment by Vignesh-Kirubanandam** [comment]
Hi,
Yes I'm using the proper dependency but there's some issue with metrics.
Logs got stopped in AWS cloudwatch.
And I saw only one log which is also "Publishing metrics into Cloudwatch
every 5m" and also "Publishing metrics into Cloudwatch every 1m" in same
actions performed.
I added duration for namespace 1m only. "cloudwatch.step" to
Duration.ofMinutes(1).toString()

implementation "io.micrometer:micrometer-registry-cloudwatch:1.9.1"
